### ISSUE TITLE
Bypass engine on defaults?

### ISSUE THREAD

**Issue by kokoko3k** [raedwulf/alsaequal#1]
I just measured alsaequal performance when using a default  (flat) preset, and i noticed that the cpu usage is the same when using an equalized profile vs using a flat profile. 
I'm talking about a low-performance atom n270 machine.
Playing an mp3 without alsaequal will use 5% cpu; using alsaequal, will lead to a 10% cpu; no matter if i use a flat profile or not.
Would it be possible to bypass the engine completely when using a flat equalizer profile? it is just a matter of convenience, so that one can set and forget alsaequal as the default pcm plugin in his .asoundrc.
Thanks.

**Comment by raedwulf** [comment]
Hey - I'm not the original author for alsaequal but I made some modifications because alsaequal had incompatible state files between x86/x86-64 so I made this repository here.  It appears the README did not make this clearer - I simply copied the original distribution and made some small changes without realising.  I'll make this clearer.

Glancing at the code, it seems that it would be possible to simply copy the values and maybe reduce the CPU usage.  I suspect the existence of the plugin will use some CPU regardless though.
